content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -6,43 +6,40 @@ weight: 10
-- ***Added*** experimental request classification filter support. This feature enables operators to configure new attributes for use in telemetry, based on request information. A primary use case for this feature is labeling of traffic by API method.  TODO: doc
-- ***Added*** Wasm extension distribution support. TODO: docs
+- ***Added*** the new [Workload Entry](/docs/reference/config/networking/workload-entry/) resource. This will allow easier configuration for non-Kubernetes workloads to join the mesh.
+- **Added** configuration for gateway topology.
+- ***Added*** experimental [mesh-wide tracing configuration API](docs/tasks/observability/distributed-tracing/configurability/). This API provides control of trace sampling rates, the maximum tag lengths for URL tags, and custom tags extraction for all traces within the mesh. 
+- **Added** experimental [request classification](/docs/tasks/observability/metrics/classify-metrics/) filter support. This feature enables operators to configure new attributes for use in telemetry, based on request information. A primary use case for this feature is labeling of traffic by API method.
 - TODO: add a statement around networking v1alpha3 API. Will it be removed in 1.6 or announce deprecation?
 
 ## Improvements
-- ***Improved*** support for [Kubernetes Ingress](https://preliminary.istio.io/docs/tasks/traffic-management/ingress/kubernetes-ingress/), adding support for reading certificates from Secrets, `pathType`, and IngressClass.
+- ***Improved*** support for [Kubernetes Ingress](/docs/tasks/traffic-management/ingress/kubernetes-ingress/), adding support for reading certificates from Secrets, `pathType`, and IngressClass.
 - ***Added*** a new `proxy.istio.io/config` annotation to override proxy configuration per pod.
-- ***Added*** support for using appProtocol to select the [protocol for a port](https://preliminary.istio.io/docs/ops/configuration/traffic-management/protocol-selection/) for Kubernetes 1.18+.
-- SDS for mixer? Not sure we need this or its an implementation detail
-- ***Improved*** Prometheus integration experience by adding standard Prometheus scrape annotations to proxies and the control plane workloads. This removes the need for specialized configuration to discover and consume Istio metrics. More details are available in the [operations guide for Prometheus](https://preliminary.istio.io/docs/ops/integrations/prometheus#option-2-metrics-merging).
-- ***Updated*** default Telemetry V2 configuration to avoid using host header to extract destination service name at gateway. This prevents unbound cardinality due to untrusted host header and implies that destination service labels are going to be omitted for request hits Blackhole and Passthrough at gateway.
+- ***Added*** support for using appProtocol to select the [protocol for a port](/docs/ops/configuration/traffic-management/protocol-selection/) for Kubernetes 1.18+.
```

Comment:
`1.18+` should be in backticks. A better option that may pass the spelling linter is "introduced in Kubernetes 1.18"

Reply:
I like `"introduced in Kubernetes 1.18"`.